.gitlab-ci.yml

```diff
@@ -1,4 +1,51 @@
-include:
-  project: 'lg/identity-gitlab'
-  ref: 'main'
-  file: '.gitlab-ci-sync.yml'
+# Jobs defined here use the idp/ci docker image from ECR by default. 
+# Images are built via the identity-devops GitLab pipeline.
+
+variables:
+
+  ECR_REGISTRY: '${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com'
+
+default:
+  image: '${ECR_REGISTRY}/dashboard/ci:latest'
```

Comment:
We can probably use one of the default Ruby images here, e.g. `public.ecr.aws/docker/library/ruby:3.3.1-bullseye` or similar

Reply:
Updated to use default ruby image